app/Console/Commands/Cloud/CloudDeleteUser.php

```
@@ -54,124 +55,141 @@ public function handle()
+            // Phase 1: Show User Overview (outside transaction)
+            if (! $this->showUserOverview()) {
+                $this->info('User deletion cancelled.');
+                $lock->release();
+
+                return 0;
+            }
+
+            // If not dry run, wrap everything in a transaction
+            if (! $this->isDryRun) {
+                try {
+                    DB::beginTransaction();
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'handle'. (undefined)

(StaticAccess)